client/codegen.yml

```diff
@@ -25,3 +27,7 @@ generates:
       scalars:
         DateTime: string
         NaiveDate: string
+        NaiveDateTime: string
+hooks:
+  afterAllFileWrite:
+    - prettier --write
```

Comment:
Will this break if someone doesn't have prettier installed? Do we need to update some docs?

Reply:
Hmm, there is something in: https://github.com/msupply-foundation/open-msupply/blob/5417317be042f067e66a13715c0f556c9312352b/CONTRIBUTING.md#L81 about it